dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/security/impl/ldap/LdapService.java

```diff
@@ -70,6 +70,15 @@ public class LdapService {
     @Value("${security.authentication.ldap.user.not-exist-action:CREATE}")
     private String ldapUserNotExistAction;
 
+    @Value("${security.authentication.ldap.ssl.enable:false}")
+    private Boolean sslEnable;
+
+    @Value("${security.authentication.ldap.ssl.trust-store:#{null}}")
+    private String trustStore;
+
+    @Value("${security.authentication.ldap.ssl.trust-store-password:#{null}}")
+    private String trustStorePassword;
```

Comment:
Can we set replace default null value with the same value in `application.yaml`? I think keep consistent is better and can also avoid NPE.

Reply:
Hi, For this part, the default will use the value in `application.yaml` not null, and even these two filed is null and is ok because truestore will use in `String.format("%s/%s", System.getProperty("user.dir"), trustStore)` works well, not throw npe, and for trustStorePassword we have a StringUtils.isNotEmpty check. WDYT?